staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -168,12 +172,18 @@ var internalPackages = []string{"client-go/tools/cache/"}
 // objects and subsequent deltas.
 // Run will exit when stopCh is closed.
 func (r *Reflector) Run(stopCh <-chan struct{}) {
-	klog.V(3).Infof("Starting reflector %v (%s) from %s", r.expectedTypeName, r.resyncPeriod, r.name)
-	wait.Until(func() {
+	fmt.Printf("Run... \n")
+	klog.V(2).Infof("Starting reflector %v (%s) from %s", r.expectedTypeName, r.resyncPeriod, r.name)
+	defer func() {
+		klog.V(2).Infof("Stopping reflector %v (%s) from %s", r.expectedType, r.resyncPeriod, r.name)
+	}()
+
+	fmt.Printf("Start to Backoff... \n")
+	wait.BackoffUntil(func() {
```

Comment:
BackoffUntil() and backoffManager are both tested so we'd skip testing reflector.Run() here